content/200-concepts/200-database-connectors/07-mongodb.mdx

```diff
@@ -3,17 +3,184 @@ title: 'MongoDB'
+| `maxIdleTimeMS`                 |         | The maximum number of milliseconds a connection can remain idle before being closed.                                                                                                                                                                                                                                           |
+| `waitQueueMultiple`             |         | The number that is multiplied by the `maxPoolSize` value to determine the amount of allowed threads that wait for the connection to become available.                                                                                                                                                                          |
+| `waitQueueTimeoutMS`            |         | The maximum numnber of milliseconds a thread can wait for a connection to become available.                                                                                                                                                                                                                                    |
+| `authSource`                    | `admin` | The database name that is associated with the authenticated credentials. If unspecified then the connection strings `database` field is used instead. If neither the `database` nor the `authSource` are set then the default value is `admin`                                                                                 |
+| `authMechanism`                 |         | Specifies the authentication method used by MongoDB to authenticate the connection. See [authMechanism docs](https://docs.mongodb.com/manual/reference/connection-string/#mongodb-urioption-urioption.authMechanism) for a run down on the available options.                                                                  |
+| `authMechanismProperties`       |         | Specifies the properties for the `authMechanism` option as a comma separated list, or a colon separated list of key-value pairs. See [authMechanismProperties docs](https://docs.mongodb.com/manual/reference/connection-string/#mongodb-urioption-urioption.authMechanismProperties) for a run down on the available options. |
+
+An example connection URL that sets:
+
+- An `ssl` connection
+- A `connectTimeoutMS`
+- And the `maxPoolSize`
+
+Would look like the following
+
+```
+mongodb://USERNAME:PASSWORD@HOST/DATABASE?ssl=true&connectTimeoutMS=5000&maxPoolSize=50
+```
+
+<Admonition type="info">
+
+For a more exhaustive list check the [connection string options docs](https://docs.mongodb.com/manual/reference/connection-string/#connection-string-options) on the MongoDB documentation.
+
+</Admonition>
+
+## Using <inlinecode>ObjectId</inlinecode>
```

Comment:
This is so specific to MongoDB, hence the overview here. It is mentioned in many other places.